README.md

```diff
@@ -34,4 +34,42 @@ Now that your local development environment is set up, you're ready to start bui
 ```bash
 cd item-seller
 cat Readme.md
-```
\ No newline at end of file
+```
+
+### Step 0.3: Installing general tools 
```

Comment:
Should this be step be in the first without git they cant do  Step 0 I suppose

Reply:
Good point, I've set it to the first step.